presto-main/src/test/java/com/facebook/presto/operator/OperatorAssertion.java

```diff
@@ -62,6 +63,21 @@ private OperatorAssertion()
 
     public static List<Page> toPages(Operator operator, Iterator<Page> input)
```

Comment:
shouldn't this be called `toPagesSafe` and the other one `toPages`? This one catches exceptions while the other one doesn't.